webserver/pkgpkr/static/main.js

```diff
@@ -32,6 +32,14 @@ $(document).ready(function () {
         $('#scoreModal').modal('hide');
     })
 
+    var page_title = window.document.title;
```

Comment:
Why do this dynamically instead of just baking the class into each page template?

Reply:
I am sorry, I do not understand your suggestion.

Reply:
It's just kind of weird to choose a header layout using JavaScript. It's more typical to have completely separate HTML templates for landing page, detail page, etc.